src/middleware/logger/index.test.ts

```diff
@@ -28,6 +28,14 @@ describe('Logger by Middleware', () => {
       return c.text(longRandomString)
     })
     app.get('/empty', (c) => c.text(''))
+    app.get('/redirect', (c) => {
+      return c.text('', 301, {
```

Comment:
You can use the `c.redirect()` though the result is the same:

```ts
return c.redirect('/empty', 301)
```

Reply:
Oh thanks. Done in https://github.com/honojs/hono/pull/3603/commits/e6b8041e3f3a12703f20dd801ffddecd1b5a28da

Reply:
👍